powersimdata/scenario/delete.py

```diff
@@ -38,29 +38,26 @@ def delete_scenario(self, confirm=True):
         self._execute_list_manager.delete_entry(scenario_id)
 
         print("--> Deleting scenario input data")
-        target = self._data_access.match_scenario_files(scenario_id, "input")
-        self._data_access.remove(target, recursive=False, confirm=confirm)
+        target = self._data_access.join(*server_setup.INPUT_DIR, f"{scenario_id}_*")
+        self._data_access.remove(target, confirm=confirm)
 
         print("--> Deleting scenario output data")
-        target = self._data_access.match_scenario_files(scenario_id, "output")
-        self._data_access.remove(target, recursive=False, confirm=confirm)
+        target = self._data_access.join(*server_setup.OUTPUT_DIR, f"{scenario_id}_*")
+        self._data_access.remove(target, confirm=confirm)
 
         # Delete temporary folder enclosing simulation inputs
         print("--> Deleting temporary folder")
-        tmp_dir = self._data_access.match_scenario_files(scenario_id, "tmp")
-        self._data_access.remove(tmp_dir, recursive=True, confirm=confirm)
+        tmp_dir = self._data_access.tmp_folder(scenario_id)
+        self._data_access.remove(tmp_dir, confirm=confirm)
 
         print("--> Deleting input and output data on local machine")
         local_data_access = LocalDataAccess()
-        target = local_data_access.join(
-            server_setup.LOCAL_DIR, "data", "**", f"{scenario_id}_*"
-        )
-        local_data_access.remove(target, recursive=False, confirm=confirm)
+        target = local_data_access.join("data", "**", f"{scenario_id}_*")
+        local_data_access.remove(target, confirm=confirm)
 
         # Delete attributes
         self._clean()
 
     def _clean(self):
         """Clean after deletion."""
-        self._data_access.close()
         self._scenario_info = None
```

Comment:
Should the parent class be `Ready`. Should we remove the `print_scenario_info` method?

Reply:
Done - moved the try/catch to the `State` implementation so it works for the delete state as well